peachjam/signals.py

```diff
@@ -75,3 +75,11 @@ def doc_deleted_update_extracted_citations(sender, instance, **kwargs):
     """Update language list on related work after a subclass of CoreDocument is deleted."""
     if isinstance(instance, CoreDocument):
         update_extracted_citations_for_a_work(instance.work_id)
+
+
+@receiver(signals.post_save)
```

Comment:
use `sender=SourceFile`

Reply:
Changed :+1:

Reply:
See https://docs.djangoproject.com/en/3.2/topics/signals/#connecting-to-signals-sent-by-specific-senders

Reply:
Much better. Thanks